pallets/parachain-staking/src/types.rs

```diff
@@ -220,7 +220,7 @@ where
 {
 	pub fn try_new(collator: AccountId, amount: Balance) -> Result<Self, ()> {
 		Ok(Delegator {
-			delegations: OrderedSet::from(vec![Stake { owner: collator, amount }].try_into()?),
+			delegations: OrderedSet::from(vec![Stake { owner: collator, amount }].try_into().unwrap()),
```

Comment:
I do not remember exactly on these lines of code, and because I am sitting on a bus I cannot compile the full code - but I am just wondering why here the `?` gots replaced by an `unwrap()`

Reply:
Yep, because of the compilation error. Otherwise, the error occurs.

```
error[E0277]: `?` couldn't convert the error to `()`
   --> pallets/parachain-staking/src/types.rs:223:84            
    |
223 |             delegations: OrderedSet::from(vec![Stake { owner: collator, amount }].try_into()?),
    |                                                                                             ^ the trait `From<std::vec::Ve
c<types::Stake<AccountId, Balance>>>` is not implemented for `()`
    |
    = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
    = help: the following other types implement trait `From<T>`: 
              <() as From<frame_system::Event<T>>>
              <() as From<pallet::Event<T>>>
              <() as From<pallet_balances::Event<T, I>>>
              <() as From<pallet_session::Event>>
              <() as From<parity_scale_codec::Compact<()>>>
              <(crypto_bigint::uint::UInt<12>, crypto_bigint::uint::UInt<12>) as From<crypto_bigint::uint::UInt<24>>>
              <(crypto_bigint::uint::UInt<14>, crypto_bigint::uint::UInt<14>) as From<crypto_bigint::uint::UInt<28>>>
              <(crypto_bigint::uint::UInt<16>, crypto_bigint::uint::UInt<16>) as From<crypto_bigint::uint::UInt<32>>>
            and 17 others
    = note: required for `Result<Delegator<AccountId, Balance, MaxCollatorsPerDelegator>, ()>` to implement `FromResidual<Result
<std::convert::Infallible, std::vec::Vec<types::Stake<AccountId, Balance>>>>`
```